README.md

```diff
@@ -47,8 +47,11 @@ You can also use `Ctrl` + `Alt` + `Shift` + `M` shortcut to open it.
 
 ### Machete file edition
```

Comment:
`Edition` isn't really used in the sense of `the action of editing`/`edycja`, it's more of `nakład, wydanie` as in `second edition of the book, revised`; maybe just `Edit machete file`